modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Adding the configuration interactively
+
+You can manage secrets and ConfigMaps you access from your function using the `func config` command.
+
+.Procedure
+
+. Run from the function project directory:
++
+[source,terminal]
+----
+$ func config -p <function-project-directory>
+----
++
+Alternatively, specify the function project directory using the `--path` or `-p` option.
+
+. Use the interactive interface to do one of the following operations:
```

Comment:
It sort of feels like these should be individual modules / procedures / use cases? It's not really clear to me what each of these does

Reply:
The only reason I've included this strange bullet point list here is to show what are all the things the user can do with the interactive utility. The utility itself is extremely straightforward, and it either only needs this much documentation or it doesn't need documentation at all. Accordingly, I've kept this "feature list", but presented it differently. Does the new presentation LGTY?

"It's not really clear to me what each of these does" Very valid. I've clarified what each of them does.

Reply:
I agree with @rh-max, I kinda like the current layout and description.